test/test-multihash.spec.ts

```diff
@@ -108,17 +107,6 @@ describe('multihash', () => {
       assert.deepStrictEqual(hash2.code, identity.code)
       assert.deepStrictEqual(hash2.bytes, hash.bytes)
     })
-
-    it('hash identity sync', () => {
```

Comment:
This is still actually sync behind the scenes though, right?  The return the of the `.digest` method is `Await<T>` which is an alias for `Promise<T> | T` - we can detect it like:

```ts
function isPromise <T> (o?: any) o is Promise<T> {
  return typeof o?.then === 'function'
}
```

then we shouldn't have to `await` the result.

```ts
it('hash identity sync', () => {
  const hash = identity.digest(fromString('test'))

  if (isPromise(hash)) {
    throw new Error('hash was promise')
  }

  assert.deepStrictEqual(hash.code, identity.code)
  assert.deepStrictEqual(identity.code, 0)
  assert.deepStrictEqual(hash.digest, fromString('test'))

  //... etc
```

Reply:
That too!